### DISCUSSION TITLE
xdg-desktop-portal-gnome & xdg-desktop-portal-hyprland

### DISCUSSION THREAD

**Discussion by carpediem29** [JaKooLit/Fedora-Hyprland#138] [General]
I would like to give Hyprland a try on Fedora - while keeping Gnome. In a few words, I wand to be able to use both
Something weird I noticed is that your script suggests to remove xdg-desktop-portal-gnome.
As far as I know multiple xdg-desktop-portal backends can be used/exist on a device. 
As mentioned by https://wiki.archlinux.org/title/XDG_Desktop_Portal : 
`When a request is made, xdg-desktop-portal will use the /usr/share/xdg-desktop-portal/DE-portals.conf file, where DE is based on the XDG_CURRENT_DESKTOP 
`
It also mentions that $XDG_CONFIG_HOME/xdg-desktop-portal/portals.conf can be used. 
Maybe the script can be adapted to use such configuration file and keep Gnome XDG Portal.

**Comment by JaKooLit** [comment]
that is just a wording before during the initial xdph

do note that on my script, it does NOT remove nor disable xdp-gnome and xdp-kde since these 2 are hard coded to the DE's

if you check the xdph.sh file, you will see which services is running. I do know that xdp-wlr and xdph does not work well enough (atleast) on my experience and the portals script is just to force start if necessary and by default its not even enabled nor set to start
